tasks/strings/strings.js

```diff
@@ -23,8 +32,9 @@ function reverseString(str) {
  * hint: please choose and appropriate method from the Math object
  */
 function centuryFromYear(year) {
-
+  return Math.round(Math.ceil(year / 100));
```

Comment:
1) `Math.ceil()` is enough here.
2) As mentioned in line 25 "if a number is passed". I cannot see this check here :)